content/es/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -0,0 +1,536 @@
+---
+title: Actualiza Objetos del API sin reemplazo (in Place) usando kubectl patch 
+description:  Usa kubectl patch para actualizar objetos del API de kubernetes sin reemplazarlos. usa strategic merge patch o JSON merge patch.
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+esta tarea muestra como utilizar `kubectl patch` para actualizar un objeto del API sin reemplazarlo.
+los ejercicios de esta tarea demuestran e uso de "strategic merge patch" y "JSON merge patch"
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+
+<!-- steps -->
+
+## Usa strategic merge patch para actualizar un Deployment
+
+Aquí está el archivo de configuración para un Deployment con dos réplicas. 
+cáda réplica es un pod que tiene un contenedor:
+
+{{% code_sample file="application/deployment-patch.yaml" %}}
```

Comment:
It seems the PR build preview is encountering [failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65fb678000d8510008d539d0) as the `{{ code_sample }} `shortcode is unable to locate the "example-ingress.yaml" for Portuguese localization.

Build Errors [(see here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65fb678000d8510008d539d0):

> 10:48:18 PM: Start building sites …
10:48:18 PM: hugo v0.121.2-6d5b44305eaa9d0a157946492a6f319da38de154+extended linux/amd64 BuildDate=2024-01-05T12:21:15Z VendorInfo=gohugoio
10:49:44 PM: ERROR [es] "deployment-patch.yaml" not found in "docs/tasks/manage-kubernetes-objects/application"
10:49:44 PM: ERROR [es] "deployment-retainkeys.yaml" not found in "docs/tasks/manage-kubernetes-objects/application"
10:49:47 PM: Total in 88736 ms
10:49:47 PM: Error: error building site: logged 2 error(s)

To address this, I suggest copying the `deployment-patch.yaml` file under [content/en/examples/application/deployment-patch.yaml](https://github.com/kubernetes/website/blob/main/content/en/examples/application/deployment-patch.yaml) to a similar location for Spanish by creating the "deployment-patch.yaml" file under "[content/es/examples/application/](https://github.com/kubernetes/website/blob/main/content/es/examples/application/)". This would likely resolve this error.